Decred Wallet/Extensions/Dcrlibwallet.swift

```diff
@@ -49,20 +49,68 @@ extension DcrlibwalletHeadersRescanProgressReport {
     }
 }
 
+
+extension DcrlibwalletBalance {
+    var dcrTotal: Double {
+        return Double(self.total / 100000000)
```

Comment:
```suggestion
        return DcrlibwalletAmountCoin(self.total)
```
Apply to the others as well.